gdal.cmake

```diff
@@ -573,6 +573,7 @@ add_subdirectory(man)
 
 # GDAL 4.0 ? Install headers in ${CMAKE_INSTALL_INCLUDEDIR}/gdal ?
 set(GDAL_INSTALL_INCLUDEDIR ${CMAKE_INSTALL_INCLUDEDIR})
+set(GDAL_INSTALL_FULL_INCLUDEDIR ${CMAKE_INSTALL_FULL_INCLUDEDIR})
```

Comment:
IMO this new variable should be introduced. Just use `CMAKE_INSTALL_FULL_INCLUDEDIR`...
... which is actually what you do.

Reply:
the point of this new variable is related to the comment "# GDAL 4.0 ? Install headers in ${CMAKE_INSTALL_INCLUDEDIR}/gdal". The variable will make it easier to track affected locations if we ever change where we install GDAL headers

Reply:
For now, I see premature pessimization/YAGNI. You could as well grep for the CMAKE variable if it is really going to be needed.
All these rarely used customization points are a barrier to sharing maintainer work and to onboarding new contributors.

Reply:
fair point. I'll do a follow-up PR to simplify this